ims/common/config.py

```diff
@@ -9,53 +9,94 @@
 __config = None
```

Comment:
>Method will create a new object each time it is called.

Not always, we can define immutable ones. Read about the ones in:

http://stackoverflow.com/questions/1318406/why-is-the-borg-pattern-better-than-the-singleton-pattern-in-python#1318444

Let me know, if you need some help, I can send some gist over next few weeks.